pkg/doc/verifiable/credential_testsuite.go

```diff
@@ -24,3 +24,17 @@ func WithPresNoProofCheck() PresentationOpt {
 		opts.disabledProofCheck = true
 	}
 }
+
+// WithPresRequireVC option enables check for at least one verifiableCredential in the VP.
+func WithPresRequireVC() PresentationOpt {
```

Comment:
may be a better name? We don't need those 'Pres' prefix since they are already 'PresentationOpt' type

Reply:
This is done to keep consistent naming of everything related to VPs. For VC, it starts from `With...`, for VP the prefix is `WithPres...`.